docs/Data/Release_Notes/Data_Release_Notes.md

```diff
@@ -49,6 +50,76 @@
 | [v2.0](Data_Release_Notes.md#data-release-20) | August 9, 2016 |
 | [v1.0](Data_Release_Notes.md#initial-data-release-10) | June 6, 2016 |
 
+## Data Release 42.0
+
+* __GDC Product__: Data
+* __Release Date__: January 2#, 2025
+
+### New Updates
+
+* New Data Sets
+    * Release of new WGS variant calling workflows for existing WGS tumor normal pairs. See the [documentation](Data/Bioinformatics_Pipelines/DNA_Seq_WGS.md) on WGS variant calling for more details on the available files. This includes data from the following pipelines:
+        * GATK4 MuTect2 - SNVs (raw and annotated VCFs)
```

Comment:
It didn't seem to register this as a bullet point - at least in terms of syntax coloring.  It's probably okay, let's just make sure to check this when it renders.